### PR TITLE
Dataset versioning

### LINKED ISSUE #2: use versioning to fetch data
https://github.com/sct-pipeline/csa-atrophy/blob/21ae3759726163504798defe6eb7b46c46414ace/csa-fetch-dataset.sh#L34

at some point we will want to reference to a specific version of the openneuro database (e.g., say v0.4).

### LINKED ISSUE #3: not sensitive if data changed on repos
with this:

https://github.com/sct-pipeline/csa-atrophy/blob/74852d00fb1290d10747c8aa86a35bf972aaff59/csa-fetch-dataset.sh#L31

if the data changed on the repos and you already have them in your local repos, then the data will not be updated (where they should)

so you probably want to find another mechanism

### PR DESCRIPTION
**Pull request by PaulBautin**
Obtaining specific subjects from the non-current version of the openneuro repository seems not possible. A reasonable approach for now is to just specify the version, download everything, and select specific subjects when running the "testing" version of the pipeline.

Question to neurostar: https://neurostars.org/t/openneuro-import-conditions/6940/2

DONE:
- modify README pipeline to integrate specific dataset version 

Fixes #2 
Fixes #3

### CODE REVIEW

**Comment by PaulBautin** [comment]
This branch has not been reviewed but I will soon close it if no objection (@jcohenadad ?) because its merge will not modify general use of "csa_atrophy".

File: README.md [hunk 1]
```diff
@@ -26,16 +25,22 @@ Download (or git clone) this repository:
...
+  ~~~
+  ./csa-fetch-dataset.sh
+  ~~~
+  - To fetch specific version of openneuro repository (version: 1.0.5, Files: 2501, Size: 7.9GB, Subjects: 248) follow instructions to set openeuro CLI: https://www.npmjs.com/package/openneuro-cli
+  ~~~
+  openneuro download --snapshot 1.0.5 ds001919 data
+  ~~~
+Run the script within the Dataset folder (script can be ran on desired subjects using flag -include)
 ~~~
 sct_run_batch -path-data data process_data.sh -jobs 2
```
Comment (jcohenadad) [inline README.md:39]: i think the script needs to be at the end of the argument list

File: README.md [hunk 2]
```diff
@@ -26,16 +25,22 @@ Download (or git clone) this repository:
...
 ~~~
 To output statistics, run in Dataset (if needed run requirements file)
```
Comment (jcohenadad) [inline README.md:41]: i will approve (out of PR scope) but could you pls open a PR to add instructions to run the requirement files (e.g. here: https://ivadomed.org/en/latest/installation.html#installation)

File: README.md (hunk 1 above)
Comment (PaulBautin) [reply README.md:39]: Not sure i understand, "Fetch" and "Run" are 2 separate actions. Should they be merged?

Comment (jcohenadad) [reply README.md:39]: sorry i wasn't clear: my comment concerns `sct_run_batch`, where the script needs to be passed as the latest argument (here you passed `-jobs 2` as latest, so it won't work)

**Pull request merged**